screens/AppNavigator/screens/Dex/DexRemoveLiquidity.tsx

```diff
@@ -0,0 +1,191 @@
+import Slider from '@react-native-community/slider'
+import { useWalletAPI } from '../../../../hooks/wallet/WalletAPI'
+import { useWhaleApiClient } from '../../../../hooks/api/useWhaleApiClient'
+
+type Props = StackScreenProps<DexParamList, 'RemoveLiquidity'>
+
+export function RemoveLiquidityScreen (props: Props): JSX.Element {
+  const navigation = useNavigation()
+
+  // this component state
+  const [tokenAAmount, setTokenAAmount] = useState<BigNumber>(new BigNumber(0))
+  const [tokenBAmount, setTokenBAmount] = useState<BigNumber>(new BigNumber(0))
+  // ratio, before times 100
+  const [percentage, setPercentage] = useState<string>('0') // for display
+  const [amount, setAmount] = useState<BigNumber>(new BigNumber(0)) // to construct tx
+
+  // gather required data
+  const tokens = useTokensAPI()
+  const { pair } = props.route.params
+  const [aSymbol, bSymbol] = pair.symbol.split('-')
+  const lmToken = tokens.find(token => token.symbol === pair.symbol) as AddressToken
+  const tokenAPerLmToken = new BigNumber(pair.tokenA.reserve).div(pair.totalLiquidity)
+  const tokenBPerLmToken = new BigNumber(pair.tokenB.reserve).div(pair.totalLiquidity)
+
+  const setSliderPercentage = useCallback((percentage: number) => {
+    // this must round down, avoid attempt remove more than selected (or even available)
+    const toRemove = new BigNumber(percentage).div(100).times(lmToken.amount).decimalPlaces(8, BigNumber.ROUND_DOWN)
+    const ratioToTotal = toRemove.div(pair.totalLiquidity)
+    // assume defid will trim the dust values too
+    const tokenA = ratioToTotal.times(pair.tokenA.reserve).decimalPlaces(8, BigNumber.ROUND_DOWN)
+    const tokenB = ratioToTotal.times(pair.tokenB.reserve).decimalPlaces(8, BigNumber.ROUND_DOWN)
+    setAmount(toRemove)
+    setTokenAAmount(tokenA)
+    setTokenBAmount(tokenB)
+    setPercentage(new BigNumber(percentage).toFixed(2))
+  }, [percentage])
+
+  const whaleAPI = useWhaleApiClient()
+  const WalletAPI = useWalletAPI()
+
+  const removeLiquidity = useCallback(() => {
+    const account = WalletAPI.getWallet().get(0) as WhaleWalletAccount
+    // TODO: add loading spinner after we have standardized design
+    constructSignedRemoveLiqAndSend(
+      whaleAPI,
+      account,
+      Number(pair.id),
+      amount
+    ).then(() => {
+      navigation.dispatch(StackActions.popToTop())
+    }).catch(e => {
+      // TODO: display error, close modal to retry/redirect
+      console.log(e)
+    })
+  }, [amount])
+
+  return (
+    <View style={tailwind('w-full h-full')}>
+      <ScrollView style={tailwind('w-full flex-col flex-1 bg-gray-100')}>
+        <View style={tailwind('w-full bg-white mt-8')}>
+          <View style={tailwind('w-full flex-row p-4')}>
+            <Text style={tailwind('flex-1')}>Amount of liquidity to remove</Text>
+            <Text style={tailwind('text-right')}>{percentage} %</Text>
+          </View>
+          <AmountSlider
```

Comment:
Slider is broken on Android
![Screenshot_20210702-101630_Expo Go](https://user-images.githubusercontent.com/25013382/124211846-67181500-db20-11eb-9d9d-45651361e472.jpg)

Reply:
Fixed